config.yaml

```diff
@@ -101,14 +101,14 @@ languages:
           links:
             - text: About Us
               link: /about
-            - text: Code of Conduct
-              link: http://scipy.github.io/devdocs/dev/conduct/code_of_conduct.html
-            - text: Developer Zone
-              link: /dev-zone
-            - text: Mailing Lists
-              link: /mailing-lists
+            - text: Community
+              link: /community
             - text: SciPy Conference
               link: https://conference.scipy.org/
+            - text: Contribute
+              link: /contribute
+            - text: Code of Conduct
+              link: http://scipy.github.io/devdocs/dev/conduct/code_of_conduct.html
```

Comment:
Is there a reason why this is not under https://scipy.org?

Reply:
AFAIK there is no reason, if we say it's better suited here than in the main doc I could move it here (or have the doc in both locations)

Reply:
Got it - i didn't realize this is how the site was structured!

Reply:
Maybe in another PR we can copy the text onto the website to have it render there as its own page.